Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -724,22 +719,22 @@ public void UpdateUser(User user)
                 throw new ArgumentNullException(nameof(user));
             }
 
-            if (user.Id.Equals(Guid.Empty) || !Users.Any(u => u.Id.Equals(user.Id)))
+            if (user.Id == Guid.Empty || !_users.TryGetValue(user.Id, out _))
```

Comment:
Noob question, why transforming the `.Equals` to `==`?

Reply:
I would assume just a side effect of the rewrite on that statement. The two should be identical after compilation (unless I'm radically misremembering C#'s == enhancements over the years)